test/Autofac.Specification.Test/Features/CircularDependencyTests.cs

```diff
@@ -94,6 +102,23 @@ public void InstancePerDependencyDoesNotAllowCircularDependencies_PropertyOwnerR
             var de = Assert.Throws<DependencyResolutionException>(() => c.Resolve<DependsByProp>());
         }
 
+        [Fact]
+        public void InstancePerDependencyDoesNotAllowCircularDependencies_PropertyOwnerResolved_WithTracerAttached()
+        {
+            var cb = new ContainerBuilder();
+            cb.RegisterType<DependsByCtor>();
+            cb.RegisterType<DependsByProp>().PropertiesAutowired(PropertyWiringOptions.AllowCircularDependencies);
+
+            var c = cb.Build();
+
+            c.AttachTrace((req, trace) =>
```

Comment:
I was looking at the build output for the PR and didn't see a trace here; is this for some other debugging? Is it dumping out to the .trx file? (Do we need it? I'm OK if it is helpful.)

Reply:
The trace capture here is to actually test that we can generate a trace capture in a call-stack depth exception scenario (to make sure that the remaining stack space detection leaves enough room for any tracing/exception catching.

Might update the asserts though to make that explicit.

Reply:
Not certain where the actual trace content ends up in CI when using `ITestOutputHelper`; it mostly only shows up in the VS Test Explorer.